VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -426,15 +501,16 @@ def archive_old_configs(self):
         if not os.path.exists(self.config_archive_folder):
             os.makedirs(self.config_archive_folder)
 
-        for root, dirs, files in os.walk(os.path.join(self._context._config_dir, '..')):
-            for file in files:
-                if file.endswith('.settings') or file == 'mrseq':
-                    src = os.path.join(root, file)
-                    dest = os.path.join(self.config_archive_folder, file)
-
-                    self.log("Copying {0} to {1}".format(src, dest))
+        # only persist latest nonquery settings file to archived settings 
+        # and prevent the accumulation of large numbers of obsolete files 
+        src = self.get_last_nonquery_config_path()
+        if src:
+            dest = os.path.join(self.config_archive_folder, 'lnq.settings')
+            if src != dest: 
+                shutil.copy2(src,dest)
 
-                    shutil.copy2(src, dest)
+        # TODO: consider saving [n].settings files in the archive folder 
+        # under versioned subfolders in addition to keeping lnq.settings 
 
     def restore_old_configs(self):
```

Comment:
Do we call this method anywhere anymore?
If we don't we can just remove this line..

Reply:
removed